core/src/main/java/com/netease/arctic/data/DefaultKeyedFile.java

```diff
@@ -215,6 +215,7 @@ public int hashCode() {
     return Objects.hash(internalFile.path());
   }
 
+  @Deprecated
```

Comment:
How about remove this method and replace it with new method?
If not, could you please add some comment to identify which version this method was deprecated and point to the new method ?

Reply:
deprecated method removed